monosi/tasks/base.py

```diff
@@ -85,6 +85,10 @@ def run(self):
             ))
 
 class MonitorsTask(ProjectTask):
+    def __init__(self, args, config):
+        super().__init__(args, config)
+        self.task_queue: List[MonitorTask] = []
```

Comment:
Making this a MonitorTask because this is a MonitorsTask specific thing and because otherwise there is no visibility on members of MonitorTask, it's a generic BaseTask